hash/msx1_cart.xml

```diff
@@ -9374,6 +9374,19 @@ kept for now until finding out what those bytes affect...
 		</part>
 	</software>
 
+	<software name="nnprowrsk" cloneof="nnprowrs">
+		<description>Nyan Nyan Pro Wrestling (Korea)</description>
+		<year>1986</year>
+		<publisher>Sammy</publisher>
+		<info name="alt_title" value="여자 레스" />
```

Comment:
Is the metadata on this correct?  The title screen says “SAMMI COMPUTER industories” and “Coconuts, 21”, and the Korean title reads “Yeoja Wres” (i.e. Women’s Wrestling).  I suspect the Japanese title would have been excised entirely in Korea because of media laws restricting use of Japanese language.

Reply:
That is indeed incorrect, thanks.
I changed the publisher to 'Sammi Computer'; at the time there were 2 Sammi Computer companies in Korea. I think it may have been this Sammi Computer: http://hardcoregaming101.net/korea/part1/company-sammi.htm (timeframe and main systems somewhat match).